test/Conductors.Tests/RepositoryConductorTests/IRepositoryMock.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AndcultureCode.CSharp.Core.Interfaces;
+using AndcultureCode.CSharp.Core.Interfaces.Conductors;
+using AndcultureCode.CSharp.Core.Interfaces.Data;
+using AndcultureCode.CSharp.Core.Interfaces.Entity;
+using Moq;
+
+namespace AndcultureCode.CSharp.Conductors.Tests.RepositoryConductorTests
+{
+
```

Comment:
nitpick (non-blocking): Extra line spacing

Reply:
will fix in next commit